daemon/k8s_watcher.go

```diff
@@ -367,6 +367,10 @@ func (d *Daemon) EnableK8sWatcher() error {
 					metrics.EventTSK8s.SetToCurrentTime()
 					if oldK8sNP := k8s.CopyObjToV1NetworkPolicy(oldObj); oldK8sNP != nil {
 						if newK8sNP := k8s.CopyObjToV1NetworkPolicy(newObj); newK8sNP != nil {
+							if k8s.EqualV1NetworkPolicy(oldK8sNP, newK8sNP) {
+								return
```

Comment:
I think we need to cover this with a metric somehow to understand how much we are ignoring. Ideally we bump a metric for each event received here and have a label type = { invalid | equal | valid }

I have some ideas here. Happy to work on this after we merge this PR.